gateway/1.4/modules/ROOT/pages/flex-local-view-logs.adoc

```diff
@@ -0,0 +1,39 @@
+= Viewing Logs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway maintains standard output logs named `flex-gateway-fluent`. Use the `flex-gateway-fluent` standard output logs for quick logging checks where an additional logging output is not needed, such as ensuring an external service is running or API requests are reaching Flex Gateway.
+
+You do not need to configure log output to `flex-gateway-fluent` for them to be visible. However, if you want Access Logs to appear in `flex-gateway-fluent` logs, you must apply a Message Logging policy to the APIs you want to monitor. To apply a Message Logging policy, see xref:flex-local-third-party-logs-config.adoc#configure-access-logs[Configuring Access logs].
+
+As well as the `flex-gateway-fluent` logs, you can configure additional Fluent Bit logging outputs. Configuring additional logging enables you to apply a custom logging output and customize the log level info. To configure additional logging, see xref:flex-local-third-party-logs-config.adoc[].
+
+To view additional log outputs, find the appropriate method to view them. The following commands only display the `flex-gateway-fluent` logs.
+
+== View Logs
+
+To view the standard output logs, run the following commands:
```

Comment:
I assume they run one command, not all. 
Maybe "run one of the following commands"?